### PR TITLE
Feature/rails 7 and adding features

### PR DESCRIPTION
**Pull request by asedge**
- Add support for Rails 7 and update Restforce dependency to newer version.
- Add `has_one` association. 
- Model generator enhancements: 
  - automatically add types to fields
  - sort fields alphabetically
  - add `table_name` to class
  - add optional namespace parameter so generated models can be namespaced
- Add get/set via `[]` and `[]=` for `SObject` attributes.

### CODE REVIEW

**Review by spacechurro — COMMENTED** [review]
This looks slick. I'm not too savvy on how CHANGELOG.md normally gets updated, but what if we add in a bullet point summary of the public facing changes for `0.9.0` in there and then we can also include that same summary in the description of this PR? Then we should all our bases covered for where to look for info on the new gem version.

**Comment by asedge** [comment]
> This looks slick. I'm not too savvy on how CHANGELOG.md normally gets updated, but what if we add in a bullet point summary of the public facing changes for `0.9.0` in there and then we can also include that same summary in the description of this PR? Then we should all our bases covered for where to look for info on the new gem version.

Makes sense. I'll see if I can update the `CHANGELOG` today.

**Comment by asedge** [comment]
@spacechurro Updated the changelog and description.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Not released
 
+## 0.9.0
+
+*  Add support for Rails 7 and update Restforce dependency to newer version. Add `has_one` association. Model generator enhancements: automatically add types to fields, sort fields alphabetically, add `table_name` to class, and add optional namespace parameter so generated models can be namespaced. Add get/set via `[]` and `[]=` for `SObject` attributes. ([#3][])
```
Comment (spacechurro) [inline CHANGELOG.md:7]: Let's split these sentences into separate bullets like you have in the PR description and we should be good to go.

**Review by spacechurro — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (asedge) [reply CHANGELOG.md:7]: Roger. I figured since they all pointed back to the same PR it made more sense to do it this way.

Comment (asedge) [reply CHANGELOG.md]: @spacechurro all done.

**Review by spacechurro — APPROVED** [review]
Ooooh. Even got some PR links in there too. Looks great. Thanks!

**Pull request merged**